clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/provider/agent/IamRoleCachingAgent.java

```diff
@@ -134,6 +133,23 @@ private void logUpcomingActions(
     return evictionsByKey;
   }
 
+  protected String getIamRegion() {
+    // sample a region from the account in case the default won't work
+    String testRegion =
+        !account.getRegions().isEmpty() && account.getRegions().get(0) != null
+            ? account.getRegions().get(0).getName()
+            : "";
+
+    if (!testRegion.equals("")
```

Comment:
Since it's a string, we can also use StringUtils.isBlank(testRegion) here - it checks null, empty or blank spaces.